src/humanPose/draw.js

```diff
@@ -1454,7 +426,11 @@ function toggleAnalyzerSettingsUI() {
  * @param {boolean} visible - whether to show or not
  */
 function setHumanPosesVisible(visible) {
```

Comment:
This would be better across all HPAs/analytics tools, it is in the top menu bar. In terms of broad semantics, toggle for 3D video in that menu also switches on/off for all avatars